debian/billboard.aegis

```diff
@@ -1,4 +1,17 @@
 <aegis>
+    <provide>
+        <credential name="control" />
+        <credential name="access" />
+        <dbus name="io.thp.billboard" own="control" bus="system">
```

Comment:
yep! i originally used the system bus since its generally more reliable, but i didnt realize that the only reason it worked was my /etc/dbus-1/system.conf.
apparently, i forgot to change it everywhere